okhttp/src/main/java/okhttp3/RealCall.java

```diff
@@ -65,6 +66,16 @@ protected RealCall(OkHttpClient client, Request originalRequest) {
     }
   }
 
+  private void captureCallStackTrace() {
+    StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
```

Comment:
Checking the log level seems reasonable. Should it be `INFO` or `FINE`? A quick experiment on Android was showing me `INFO` was evaluating to true.